migration/incremental/025-tag-map-storage-type.sql

```diff
@@ -0,0 +1,23 @@
+-- We have to mutate the catalog for two reasons:
+-- 1. ALTER TYPE tag_map SET (STORAGE = EXTENDED); is not supported on PG12
+-- 2. We can not ALTER hypertables that have compression enabled.
+
+-- tag_v
+UPDATE pg_catalog.pg_type AS t 
+SET typstorage = 'x' 
+WHERE t.oid = '_ps_trace.tag_v'::regtype::oid;
+
+UPDATE pg_catalog.pg_attribute AS a
+SET attstorage = 'x'
+WHERE a.attrelid = '_ps_trace.tag'::regclass::oid
+  AND a.atttypid = '_ps_trace.tag_v'::regtype::oid;
```

Comment:
`tag_v` is never stored, so this is essentially a no-op.

Reply:
Actually I wasn't confused. We have two distinct types:
- `ps_trace.tag_v` which is used in `tag` table
- `_ps_trace.tag_v` which shouldn't ever be used anywhere.

Reply:
I don't think that's correct. Yes, the two types do exist. `ps_trace.tag_v` _was_ used in the `_ps_trace.tag` table (in `012-tracing.sql`), but it was replaced by `_ps_trace.tag_v` in `015-tracing-redesign.sql`.